### PR TITLE
Ta 2908 office hero section

### PR DESCRIPTION
**Pull request by cyber-decker**
Adding hero tests

### CODE REVIEW

File: cypress/integration/tests/office-page.test.js [hunk 1]
```diff
@@ -174,6 +175,73 @@ describe("District Office Page", function () {
...
+                    .contains("Learn More")
+                    .and('has.attr', "href", expectedButtonUrl)
+                cy.get("div")
+                    .should('have.css', 'background-image', `url("${Cypress.config().baseUrl}${expectedImageUrl}")`)
+                    .and('have.attr', 'aria-label', expectedAltText)
+            })
+        })
+
+        it("does not display a button when there is no banner link", function () {
+            delete this.GenericOffice.bannerImage.link.url
```
Comment (amyd11) [inline cypress/integration/tests/office-page.test.js]: This delete command is a cool thing to use- nice and readable

File: cypress/integration/tools/office-page-tests.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+describe("Office Page", function () {
+    beforeEach(function () {
+        cy.fixture("office/6386.json").as("MockOffice")
+    })
+    describe("Hero section tests", function () {
```
Comment (amyd11) [inline cypress/integration/tools/office-page-tests.js]: It doesn't look like this is actually testing anything. I think we can delete it, I'm assuming it was uploaded accidentally?

Comment (amyd11) [reply cypress/integration/tools/office-page-tests.js]: And subsequently delete the fixture `6386.json` since we're not using it for anything else

Comment (cyber-decker) [reply cypress/integration/tools/office-page-tests.js]: Just a tool, doesn't need tests.  This is for testing and exploring only.

**Pull request merged**